redwood-lazylayout-view/src/main/kotlin/app/cash/redwood/lazylayout/view/ViewLazyList.kt

```diff
@@ -212,16 +212,8 @@ internal open class ViewLazyList(context: Context) : LazyList<View> {
 
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
       return when (viewType) {
-        VIEW_TYPE_PLACEHOLDER -> ViewHolder.Placeholder(
-          Container(parent.context).apply {
-            layoutParams = FrameLayout.LayoutParams(MATCH_PARENT, WRAP_CONTENT)
-          },
-        )
-        VIEW_TYPE_ITEM -> ViewHolder.Item(
-          Container(parent.context).apply {
-            layoutParams = FrameLayout.LayoutParams(MATCH_PARENT, WRAP_CONTENT)
-          },
-        )
+        VIEW_TYPE_PLACEHOLDER -> ViewHolder.Placeholder(Container(parent.context))
+        VIEW_TYPE_ITEM -> ViewHolder.Item(Container(parent.context))
```

Comment:
How about pulling the instantiation outside of the `when` since it's common to both?